salt/minion.py

```diff
@@ -1449,6 +1449,20 @@ def get_executor(name):
             ret,
             timeout=minion_instance._return_retry_timer()
         )
+
+        # Add default returners from minion config
+        if isinstance(opts.get('return'), six.string_types):
```

Comment:
Yeah, if we want the comm-delimited version that's where it should go. We should have a known type by the time we get into the running minion.